modules/openapi-generator/src/test/java/org/openapitools/codegen/elm/ElmClientCodegenTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2019 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2019 SmartBear Software
```

Comment:
@Elewyth thanks for the PR. For new files, please remove line 3: https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license/7301#7301

Reply:
A pleasure :). removed the line.